### PR TITLE
add some basic documentation

### PR DESCRIPTION
**Pull request by tobi812**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,9 +2,37 @@
...
+
+Two Problems that can often be observed in a distributed service architecture are:
+
+A. **Too many dependencies** between services.
+
+B. No common **platform-wide definition** of a data object or business entity.
+
+Wayne approaches those problems with the idea of a _schema registry combined with a central data store_ for business 
+entities. 
+It is built on top of Django and comes with a simple API to create business entities. The integration of _json-schema_
```
Comment (lleithold) [inline README.md]: "built on top of" sound a bit weird to me, wayne is not build on top of django like for example nakadi is build on top of kafka, extending and encapsulating the functionality. for me, wayne is more like "built with" django.

File: README.md [hunk 2]
```diff
@@ -2,9 +2,37 @@
...
+
+A. **Too many dependencies** between services.
+
+B. No common **platform-wide definition** of a data object or business entity.
+
+Wayne approaches those problems with the idea of a _schema registry combined with a central data store_ for business 
+entities. 
+It is built on top of Django and comes with a simple API to create business entities. The integration of _json-schema_ 
+allows to define custom schema definitions, that will be used to validate the entities. A service that
+is "interested" in updates of a certain entity-type is also able to subscribe to _event-streams_, the default technology
```
Comment (lleithold) [inline README.md]: Why put that "interested" into quotes? :)

File: README.md (hunk 1 above)
Comment (tobi812) [reply README.md]: Totally right, I will change it :+1:

File: README.md (hunk 2 above)
Comment (tobi812) [reply README.md]: Because it feels a bit weird, that a service is interested in anything. But I couldn't come up with a better idea.

**Pull request merged**